srcjs/output_binding_html.js

```diff
@@ -107,23 +107,25 @@ function renderDependency(dep) {
   }
 
   if (dep.stylesheet) {
-    var stylesheets = $.map(asArray(dep.stylesheet), function(stylesheet) {
+    var link = $("<link rel='stylesheet' type='text/css'>");
+    $.map(asArray(dep.stylesheet), function(stylesheet) {
       var this_href = href + "/" + encodeURI(stylesheet);
+      $head.append(
+        // Force client to re-request the stylesheet if we've already seen it
+        // (without this unique param, the request might get cached)
+        link.attr("href", restyle ? (this_href + "?restyle=" + new Date().getTime()) : this_href)
+      );
+      if (!restyle) return;
       // If a styleSheet with this href is already active, then disable it
-      var re = new RegExp(this_href + "$");
+      var re = new RegExp(this_href.split("/").join("\\/") + "$");
```

Comment:
Instead of the split and join, maybe use encodeURIComponent or encodeURI?

Reply:
What I'd really want is a way to escape all the regex special characters, but I think I've found a reasonable to go about the same thing without regexing